modules/store/src/tokens.ts

```diff
@@ -24,6 +24,11 @@ export const _STORE_REDUCERS = new InjectionToken(
 export const _FEATURE_REDUCERS = new InjectionToken(
   '@ngrx/store Internal Feature Reducers'
 );
+
+export const FEATURE_NAME = new InjectionToken(
+  '@ngrx/store Internal Feature name'
```

Comment:
This shouldn't be internal?

Reply:
Why does this need to be separate if the `key` is included in the `StoreFeature<T, V>`?